lang/syn/src/codegen/program/handlers.rs

```diff
@@ -464,7 +465,7 @@ pub fn generate(program: &Program) -> proc_macro2::TokenStream {
                                     // Load state.
                                     let mut remaining_accounts: &[AccountInfo] = accounts;
                                     if remaining_accounts.is_empty() {
-                                        return Err(anchor_lang::error::ErrorCode::AccountNotEnoughKeys.into());
+                                        return Err(anchor_lang::error::Error::from(anchor_lang::error::ErrorCode::AccountNotEnoughKeys));
```

Comment:
Can this change be undone now? (along with the other instances of this pattern)

Reply:
I can try which ones work like before but sometimes it's not able to infer now

Reply:
e.g. this fails https://github.com/project-serum/anchor/runs/5265926568?check_suite_focus=true

not sure why tbh there's only a single `From` that would make sense here

Reply:
Maybe it's it a `.into()` vs `?` problem?

Reply:
I"m confused. Is this error caused by https://github.com/project-serum/anchor/pull/1462/commits/c83e28faa367ac587215f1de2b4e0c562c24ee01 ?

Reply:
> I"m confused. Is this error caused by https://github.com/project-serum/anchor/commit/c83e28faa367ac587215f1de2b4e0c562c24ee01 ?

yes

> Maybe it's it a .into() vs ? problem?

How would that be problematic?

Reply:
the only competing From impl is `u32` but seems like the compiler should be smart enough to infer the correct one